content-src/components/DiscoveryStreamBase/_DiscoveryStreamBase.scss

```diff
@@ -13,4 +13,9 @@
 
     $columns: $columns - 1;
   }
+
+  .column {
```

Comment:
Does this want to be .ds-column?  (And maybe the numbered ones also want the prefix).

Reply:
Changing layout to ds-layout too.